src/plugins/intel_cpu/src/nodes/mvn.cpp

```diff
@@ -1973,989 +161,122 @@ void MVN::initSupportedPrimitiveDescriptors() {
-    }
-
-    if (canUseAclExecutor) {
-        std::vector<MemoryDescPtr> srcMemoryDescs;
-        for (size_t i = 0; i < getParentEdges().size(); i++) {
-            srcMemoryDescs.push_back(getSrcMemoryAtPort(i)->getDescPtr());
-        }
-        std::vector<MemoryDescPtr> dstMemoryDescs;
-        dstMemoryDescs.push_back(getDstMemoryAtPort(0)->getDescPtr());
-
-        auto selectedPD = getSelectedPrimitiveDescriptor();
-        aclExecPtr = selectedPD->getExecutorFactoryAs<MVNExecutorFactory>()->makeExecutor(mvnAttrs,
-                                                                                          srcMemoryDescs,
-                                                                                          dstMemoryDescs,
-                                                                                          {});
-        selectedPD->setImplementationType(aclExecPtr->getImplType());
-
-        return;
-    }
-
-    MVNKey key = {mvnAttrs, dnnl::primitive_attr()};
-    setPostOps(key.attr, true);
-
-    auto builder = [&](const MVNKey& key) -> std::shared_ptr<MVNExecutorBase> {
-        std::shared_ptr<MVNExecutorBase> executor;
-        if (mayiuse(cpu::x64::sse41)) {
-            executor = std::make_shared<MVNJitExecutor>(key.mvnAttrs, key.attr);
-        } else {
-            executor = std::make_shared<MVNRefExecutor>(key.mvnAttrs);
-        }
-        return executor;
-    };
-
-    auto cache = context->getParamsCache();
-    auto result = cache->getOrCreate(key, builder);
-    execPtr = result.first;
-}
-
-void MVN::transformTo5DCase(const VectorDims& shape) {
-    size_t rank = shape.size();
-    // for 1 and 2 rank, if initAcrossChannels_ is true, adjust shape to fully vectorize under unified 5d procedure.
-    // otherwise there are not enough data in spatial dimension to process in one kernel.
-    switch (rank) {
-    case 1:  // C
-        if (mvnAttrs.initAcrossChannels_) {
-            shape5D = {1, 1, 1, 1, shape[0]};
-            mvnAttrs.execAcrossChannels_ = false;
-            break;
-        } else {
-            shape5D = {1, shape[0], 1, 1, 1};
-            break;
-        }
-    case 2:  // NC
-        if (mvnAttrs.initAcrossChannels_) {
-            shape5D = {1, shape[0], 1, shape[1], 1};
-            mvnAttrs.execAcrossChannels_ = false;
-            break;
-        } else {
-            shape5D = {shape[0], shape[1], 1, 1, 1};
-            break;
-        }
-    case 3: {
-        shape5D = {shape[0], shape[1], 1, shape[2], 1};
-        break;
-    }
-    case 4: {
-        shape5D = {shape[0], shape[1], 1, shape[2], shape[3]};
-        break;
-    }
-    case 5: {
-        shape5D = {shape[0], shape[1], shape[2], shape[3], shape[4]};
-        break;
-    }
-    default: {
-        THROW_CPU_NODE_ERR("doesn't support planar layout with rank: ", shape.size());
-    }
+        OPENVINO_THROW("Preferable primitive descriptor is not set.");
     }
+    executor = createExecutor();
```

Comment:
The flow has changed a bit. Please align with fc / convoluiton nodes.
Now we create executor in scope of CreatePrimitive and update it in scope of prepareParams